src/Model/TrainCarCardDeck.java

```diff
@@ -18,6 +18,31 @@ public iCard draw() {
         return card;
     }
 
+    public iCard drawTopCard(){
```

Comment:
I made a function to get a random card on the deck class. This one would give a bunch of cards of the same type in a row because the json is organized. It's called "draw" in the TrainCarCardDeck class.